src/js/components/molecules/article-table/article-table.vue.html

```diff
@@ -0,0 +1,3 @@
+<!-- @format -->
+
+<div class="article-table" v-html="table" />
```

Comment:
locally i changed this to `<table` and it worked as well without adding a table in a table.

Reply:
I'll give it a go…